src/components/HendelseFilterPanel.tsx

```diff
@@ -250,6 +252,19 @@ export function HendelseFilterPanel({ personRegister }: Props) {
       type: ActionType.SetSelectedHendelseType,
       selectedHendelseType: newFilterState,
     });
+    const shouldLogToAmplitude = !!checkboxElements.find(
+      (checkbox) => checkbox.hendelse === value && !checkbox.isChecked
+    );
```

Comment:
Tanken er at vi kun logger events der man checker en checkbox, ikke der man unchecker den. 
Da kan vi telle hvilke filtere de bruker, men vi får ikke vite om man bruker to filtere sammen eller ikke.

Reply:
Forslag: Bruke `.some()`, siden man bare vil sjekke om et element som matcher et predikat eksisterer og ikke gjøre noe mer med elementet :)

Reply:
Ja stemmer, riktig! Takk 🙇🏼‍♂️